SeafarersAPI/Controllers/SubmissionsController.cs

```diff
@@ -20,7 +20,7 @@ public class SubmissionsController : ControllerBase
         [HttpGet]
         public List<Submission> Get()
         {
-            return submissionService.GetSubmissions();
+            return submissionService.GetSubmissions(0, 50).ToList();
```

Comment:
Add this because we want pagination before the submissions get too big.

Reply:
Good point.